app/views/dashboard/_alphabetical_collections_and_document_sets.html.slim

```diff
@@ -0,0 +1,16 @@
+
+    .collections
+      -collections_and_document_sets.each do |cds|
+        -if cds.show_to?(current_user)
+          -snippet = truncate(strip_tags(cds.intro_block), length: 300, separator: ' ') || ''
+          -owner_link = link_to cds.owner.display_name, user_profile_path(cds.owner)
+          .collection
+            -unless cds.picture.blank?
+              .collection_image
+                =image_tag(cds.picture_url(:thumb), alt: cds.title)
+            .collection_details
+              h4.collection_title =link_to cds.title, collection_path(cds.owner, cds)
+              -unless snippet.empty?
+                p.collection_snippet =snippet
+              .collection_summary
+                span =="Owned by #{owner_link}"
```

Comment:
Though the diff makes this look like a new file, it is just a rename
- from: `app/views/dashboard/_collection_index.html.slim`
- to: `app/views/dashboard/_alphabetical_collections_and_document_sets.html.slim`

Reply:
Looks good.